### DISCUSSION TITLE
Porous jump

### DISCUSSION THREAD

**Discussion by torodecamargue** [nextfoam/baram#74] [General]
Hello,

I have updated Baram on Windows to the version v24.2.0.

I use the porous jump feature to simulation a perforated sheet inside an air duct. 
Since the update I am not able to select the coupled boundary between the 2 regions to be a pressure jump Boundary.

On the last version it was possible.

Best regards

Julien

**Comment by kdg5424** [comment]
Hello Julien.

I want you to check one more time.
There is porous jump boundary, as like the last version. (If it's correct ,pressure jump what you told means porous jump.)

![image](https://github.com/nextfoam/baram/assets/133930861/cad558f5-c16c-47bb-806b-a42df347cf60)
![image](https://github.com/nextfoam/baram/assets/133930861/c4b3c1ee-5ea7-42e6-a24e-d642f0a6b2ce)

**Comment by kdg5424** [comment]
Hello Julien.
I think I misunderstood your question.
You said that you can not select the porous jump between 2 regions.
After I check it, I will tell you again.
Thanks.

P.S.) It's my linkedIn profile. So if you have the linkedIn profile, I wish you to add me as a friend.
https://www.linkedin.com/in/donggyu-kim-072b51231/

Regards
Donggyu

**Comment by kdg5424** [comment]
Hello Julien. This is Donggyu.

We check the porous jump between 2 regions.
And it is correct that **porous jump is not configured between 2 regions.** The previous version was incorrect.
There could not be momentum exchange, pressure drop between fluid-solid.
And if you want momentum exchange, pressure drop and etc between fluid-fluid, then **you should make just one region**.
So, could you tell us what is your situation with more detail?
There maybe the things we haven't thought in your situation. And maybe we could help you.
And if you're ok, add me as a friend in linkedin.

Thanks

Regards
Donggyu kim

**Comment by torodecamargue** [comment]
Hello Donggyu,

Thank you for your support. I don't have linkedIn profile but I will create one this week end.

Yes I mean porous jump, not pressure jump. 
I need to calculate the flow passing trough a perforated plate inside an air duct. How can I manage to have only 1 region in this case ? During the mesh operations (2.Region step) I have to define a point in the region, and the mesh is only created in the region I choose.

How to define the porous jump boundary in meshing operation, an "interface" but it creates 2 regions ? If I choose "none" only 1 region is created, but I don't have access to the boundary in BaramFlow. If I define it as a "boundary " the mesh is create in only 1 part of the case...

![image](https://github.com/nextfoam/baram/assets/75060318/497c8556-25c4-4962-b1df-b0e254f404a9)

![image](https://github.com/nextfoam/baram/assets/75060318/0645dc9e-bb35-4a26-af33-41481aeb0642)

![image](https://github.com/nextfoam/baram/assets/75060318/815d1d0e-f9d0-4253-b410-89b6a203373f)

Best regards 

Julien

**Comment by jiban** [comment]
Hi Julien,
You can set the geometry type of porous jump to *interface* like following in BaramMesh.
Please note that you **should NOT** check *Non-Conformal* nor *Inter-Region*.
( *Inter-Region* option is used to separate regions )

![interface](https://github.com/nextfoam/baram/assets/1674177/16e33141-e084-4a25-b1e2-20b6b1775d81)


After *snap* in *5.Snap* step, you may found coupled boundaries like following.
You can use these master/slave pair of boundaries as porous jump couple in BaramFlow.

![masterSlave](https://github.com/nextfoam/baram/assets/1674177/8dc2b229-29d0-4227-933e-335a2eea9d46)


By the way, you need to specify only one region point in *2.Region* step since there is only one region.

Best regards,
Jake

**Comment by torodecamargue** [comment]
Hello,

Thank you for your help. I select only "interface" and it is ok.

Now I can assign correctly the porous jump. I calculate coefficient according to https://holzmann-cfd.com/community/blog-and-tools/darcy-forchheimer. with my own perforated plates pressure drops.

![image](https://github.com/nextfoam/baram/assets/75060318/f4c57cf4-3661-45ef-bbcf-02a178082ad1)

But I don't see any jump in pressure with paraview and no additional pressure at the inlet with the monitoring. I modify the coefficients, thickness of the porous jump, signs... I have always the same results, same convergence curve. Maybe when I record the BaramFlow file, it is not updated? 

I look into the "0" folder of the case I don't see anywhere the Inertial and Darcy coefficients that I use in the GUI, only "cyclic" condition.

![image](https://github.com/nextfoam/baram/assets/75060318/07da1c19-038a-4a22-8f92-77cb6320eef5)

My porous jump boundary is a single boundary, maybe it is a problem, 2 boundaries are needed to define the thickness ? 

Best regards

Julien

**Reply by jiban** [reply]
Dear Julien,
You need to see `p_rgh` file rather than `p` file to check the Inertial and Darcy coefficients since the calculation is base on *p_rgh* value in BARAM.

**Comment by torodecamargue** [comment]
Ok now it works and gives the results close to the analytics formula. I use only 1 processor, I have problem for running parallel cases.
I have to write thickness = 1 m and coefficient match with the calculated pressure drops! 26 Pa calculated and 26 Pa with Baram! Perfect

![image](https://github.com/nextfoam/baram/assets/75060318/88c29fb1-7a5b-44d9-bb94-8c1eeeeb5074)

Best regards

**Comment by kdg5424** [comment]
Hello Julien.
Could you let me know what's the problem with parallel running?
Because in my PC, there is no problem with running parallel case.
So could you let me know the detail of the problem?

Thanks

Donggyu

**Comment by kdg5424** [comment]
Ah, I found the issue with #75 . I will write down the soultion at there.

**Comment by torodecamargue** [comment]
Hello,

The problem of matching tolerance with porous jump is not fixed with the last version of Baram. 

Actually it is worse with the v26.3 version, because increasing the tolerance manually in the constant folders doesn't fix the problem. Need to run with only 1 processor... Do you know how to fix it in my case?

Best regards

Julien
